### ISSUE TITLE
Ensure all activities/tasks have <answer>s

### ISSUE THREAD

**Issue by siwelwerd** [TeamBasedInquiryLearning/precalculus#192]
We created the instructor version midway through writing, so we do not currently have `<answer>`s for all of the activities.

**Comment by siwelwerd** [comment]
To list these, run `xpath -e '//activity[not(task) and not(answer)]' source/*/*.ptx` to find activities with no tasks and no answer, and `xpath -e '//task[not(answer)]' source/*/*.ptx` to find tasks that don't have an answer.

**Comment by StevenClontz** [comment]
It'd be great to have a linter that runs in GitHub to check for these things. Perhaps if we had one, it'd be in scope for the EditAThon to make this enhancement for LA and Calc too?

**Comment by siwelwerd** [comment]
I don't know what a linter is, but I can think of a few things that PreTeXt doesn't check for that I'd appreciate our action checking for.  The other big one that comes to mind is `@xml:id`s for certain things (e.g. images; see https://github.com/TeamBasedInquiryLearning/calculus/issues/78)

>it'd be in scope for the EditAThon to make this enhancement for LA and Calc too?

Do you mean copy the action over to those books, or to create instructor versions for them which contain `<answer>`s?

**Comment by StevenClontz** [comment]
A linter is a tool that checks that certain preferences in a software product are satisfied. So theoretically we could add preferences that images have `xml:id`s, that `task`s have `answer`s, etc.

> Do you mean copy the action over to those books, or to create instructor versions for them which contain <answer>s?

To create instructor versions with `answer`s, so that our linter won't balk at the lack thereof.

**Comment by siwelwerd** [comment]
In the hopes that someone tackles this during the Edit-A-Thon, here is a list of how many occurrences in the various sections

```
(base) drew@Drews-MBP-2 precalculus % xpath -e '//task[not(answer)]' source/*/*.ptx 2>&1 | grep Found           
Found 26 nodes in source/01-EQ/01.ptx:
Found 1 nodes in source/01-EQ/03.ptx:
Found 18 nodes in source/01-EQ/04.ptx:
Found 50 nodes in source/01-EQ/05.ptx:
Found 26 nodes in source/01-EQ/06.ptx:
Found 1 nodes in source/01-EQ/07.ptx:
Found 1 nodes in source/02-FN/03.ptx:
Found 4 nodes in source/02-FN/04.ptx:
Found 17 nodes in source/02-FN/06.ptx:
Found 4 nodes in source/03-LF/01.ptx:
Found 5 nodes in source/03-LF/02.ptx:
Found 6 nodes in source/03-LF/06.ptx:
Found 18 nodes in source/04-PR/01.ptx:
Found 16 nodes in source/04-PR/02.ptx:
Found 1 nodes in source/04-PR/04.ptx:
Found 29 nodes in source/04-PR/05.ptx:
Found 33 nodes in source/04-PR/06.ptx:
(base) drew@Drews-MBP-2 precalculus % xpath -e '//activity[not(task) and not(answer)]' source/*/*.ptx 2>&1 | grep Found
Found 1 nodes in source/01-EQ/05.ptx:
Found 1 nodes in source/04-PR/05.ptx:
Found 1 nodes in source/04-PR/06.ptx:
```

**Comment by tdegeorge** [comment]
This might also be a good time to make sure we are all doing it consistently....like some answers have just the letter. Some answers have the letter and the answer written out
@kathypinzon @AbbyANoble

**Comment by kathypinzon** [comment]
I vote just letter.... but if it already has letter and answer I don't want to edit that now

**Comment by siwelwerd** [comment]
We should follow the guidance in the [Style Guide](https://github.com/TeamBasedInquiryLearning/library/wiki/Style-Guide).

**Comment by tdegeorge** [comment]
> Style Guide

I guess we already talked about this! :-p I don't mind going through and checking the ones that already have answers. I would rather it is consistent before we mark them as complete in our google sheet!

**Comment by siwelwerd** [comment]
Update: Almost done!  PR3 and PR6 look to be devoid of answers, and then a few others missing here and there.  Just re-ran my script to check:
```
Found 1 nodes in source/01-EQ/07.ptx:
Found 4 nodes in source/03-LF/01.ptx:
Found 5 nodes in source/03-LF/02.ptx:
Found 6 nodes in source/03-LF/06.ptx:
Found 39 nodes in source/04-PR/03.ptx:
Found 34 nodes in source/04-PR/06.ptx:
```

**Comment by siwelwerd** [comment]
I think this will be fixed after #418, #458, and #459 are merged in.
